File: exercises-python/2. Problemas de estructura condicional/12.py
'''
12.Desarrolle el programa que lea un número entero en el intervalo
de 1 a 7 correspondiente a un día de la semana, y determine el
nombre del día. Considere 1 - lunes, 2 – martes, …
'''

import os
os.system("cls")

#Datos
num = int(input("Número (del 1 al 7): "))

#Salida
if num == 1: print("Lunes")
elif num == 2: print("Martes")
elif num == 3: print("Miercoles")
elif num == 4: print("Jueves")
elif num == 5: print("Viernes")
elif num == 6: print("Sábado")
elif num == 7: print("Domingo")

File: exercises-cpp/3. C++/2. Condicionales/12.cpp
#include <iostream>
#include <cstdio>
#include <cmath>
using namespace std;

int main()
{
    system("cls");
    
    int num;

    printf("Número (del 1 al 7): "); cin>>num;

    if (num == 1) printf("Lunes");
    else if (num == 2) printf("Martes");
    else if (num == 3) printf("Miercoles");
    else if (num == 4) printf("Jueves");
    else if (num == 5) printf("Viernes");
    else if (num == 6) printf("Sábado");
    else if (num == 7) printf("Domingo");
}